.codex-plugin/ycc/skills/compatibility-audit/scripts/audit-install-assumptions.sh

```
@@ -0,0 +1,237 @@
+record() { CT+=("$1"); CI+=("$2"); CS+=("$3"); CD+=("$4"); }
+
+json_ok() { set +e; python3 -m json.tool "$1" >/dev/null 2>&1; local r=$?; set -e; return $r; }
+
+# ---------------------------------------------------------------------------
+# Claude checks
+# ---------------------------------------------------------------------------
+run_claude_checks() {
+  local mkt="${REPO_ROOT}/.codex-plugin/marketplace.json"
+  local plugin="${REPO_ROOT}/ycc/.codex-plugin/plugin.json"
+
+  json_ok "$mkt" && record "claude" "C1" "PASS" "" \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)